gbpservice/neutron/services/grouppolicy/drivers/oc_resource_mapping.py

```diff
@@ -0,0 +1,271 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import netaddr
+
+from neutron.common import log
+from neutron.openstack.common import jsonutils
+from neutron.openstack.common import log as logging
+from neutron.plugins.common import constants as pconst
+from neutron.common import exceptions as n_exc
+from neutron.extensions import securitygroup as ext_sg
+from oslo.config import cfg
+
+from gbpservice.neutron.services.grouppolicy.common import constants as gconst
+from gbpservice.neutron.services.grouppolicy.common import exceptions as exc
+from gbpservice.neutron.extensions import group_policy as gp_ext
+
+from gbpservice.neutron.services.grouppolicy.drivers import resource_mapping
+
+LOG = logging.getLogger(__name__)
+
+
+class OCResourceMappingDriver(resource_mapping.ResourceMappingDriver):
+    """Resource Mapping driver for Group Policy plugin.
```

Comment:
Comment should include description of specific functionality offered by the driver